.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 100
+ignore = F401, E241, W504
```

Comment:
As discussed, maybe just suppress these locally in your file

Reply:
done in https://github.com/ll7/paf22/pull/170/commits/48f63365232b9939d859252c7f5d7560637d498f